### PR TITLE
0.9

### PR DESCRIPTION
**Pull request by nixtrace**
Features:
- Global refactor
- Radix
- Go mod
- Redis auth support
- Dump to file
- Load from dump file
- TTL sync
- Silent mode
- Signal handling
- Tests
- CI
- Docker dev setup
- Docker examples

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 rump-*
+main
+*.dump
```
Comment (JoN1oP) [inline .gitignore]:
```suggestion
*.rump
```

File: pkg/redis/redis_test.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+func TestMain(m *testing.M) {
+	setup()
+	code := m.Run()
+	teardown()
+	os.Exit(code)
+}
+
+func TestTTL(t *testing.T) {
+	// key := "key1"
```
Comment (JoN1oP) [inline pkg/redis/redis_test.go]: drop or fix this test?

Comment (nixtrace) [reply pkg/redis/redis_test.go]: drop, thanks!

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+```
+You can run `rump` in a container following the [Dockerfile.example](/infra/Dockerfile.example).
+
+## Mentions
+
+- [Sticker Mule Blog](https://www.stickermule.com/blog/introducing-rump)
+- [The Changelog](http://email.changelog.com/t/ViewEmail/t/13CBF627BB99BB74/)
+- [Go Weekly](http://golangweekly.com/issues/138)
+- [DB Weekly](http://dbweekly.com/issues/132)
+- [Redis Weekly](http://redisweekly.com/archive/172.html)
```
Comment (JoN1oP) [inline docs/README.md]: 404

**Review by JoN1oP — APPROVED** [review]
:+1:

**Pull request merged**